app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/auth/AuthCodeConverter.kt

```diff
@@ -5,11 +5,12 @@ import androidx.compose.runtime.collectAsState
 import com.bottlerocketstudios.compose.auth.AuthCodeState
 
 @Composable
-fun AuthCodeViewModel.toState() = AuthCodeState(
+fun AuthCodeViewModel.toState(showToolbar: (Boolean) -> Unit) = AuthCodeState(
```

Comment:
@BottleRocket-Colin Would be nice to also name the inner Boolean argument like so for additional context:

```suggestion
fun AuthCodeViewModel.toState(showToolbar: (show: Boolean) -> Unit) = AuthCodeState(
```